ts/main.ts

```diff
@@ -42,7 +43,99 @@ $allForm.addEventListener('submit', (event: Event) => {
   data.entries.unshift(entryObject);
 
   writeData();
+  viewSwap('entries');
+  const resultRender = renderEntry(entryObject);
+  ulParent.prepend(resultRender);
 
   $placeholderImg.setAttribute('src', 'images/placeholder-image-square.jpg');
   $allForm.reset();
 });
+
+function renderEntry(entry: Entries): HTMLLIElement {
+  const $liChild = document.createElement('li');
+  $liChild.setAttribute('class', 'first-entry');
+  const $divRow = document.createElement('div');
+  $divRow.setAttribute('class', 'row');
+  const $divColumn = document.createElement('div');
+  $divColumn.setAttribute('class', 'column-half');
+  const $image = document.createElement('img');
+  $image.setAttribute('src', entry.url);
+  $image.setAttribute('alt', 'placeholder image');
+  $image.setAttribute('id', 'textimg');
```

Comment:
You can't give elements the same `id`, so you should delete this. This would give every single `img` the exact same `id`